modules/ROOT/nav.adoc

```diff
@@ -1,99 +1,20 @@
-  *** xref:transform-dataweave-xml-reference.adoc[Transform XML Reference]
- ** xref:try-scope-concept.adoc[Try Scope]
-  *** xref:try-scope-xml-reference.adoc[Try Scope XML Reference]
- ** xref:until-successful-scope.adoc[Until Successful Scope]
-* xref:mule-app-strategies.adoc[General Configurations]
- ** xref:configuring-properties.adoc[Configuring Properties]
-  *** xref:mule-app-properties-to-configure.adoc[Configure Property Placeholders for Mule Apps]
-  *** xref:mule-app-properties-system.adoc[System Properties]
-  *** xref:custom-configuration-properties-provider.adoc[Custom Configuration Properties Provider]
-  *** xref:deploying-to-multiple-environments.adoc[Configure Environment Properties]
- ** xref:error-handling.adoc[Error Handling]
-  *** xref:mule-error-concept.adoc[Mule Errors]
- ** xref:streaming-about.adoc[Streaming in Mule Apps]
-  *** xref:streaming-strategies-reference.adoc[Streaming Strategies Reference]
- ** xref:shared-resources.adoc[Shared Resources]
-  *** xref:setting-environment-variables.adoc[Setting Environment Variables]
- ** xref:target-variables.adoc[Enrich Data with Target Parameters]
- ** xref:reconnection-strategy-about.adoc[Reconnection Strategies]
-  *** xref:reconnection-strategy-reference.adoc[Reconnection Strategy Reference]
- ** xref:redelivery-policy.adoc[Redelivery Policy]
- ** xref:mule-server-notifications.adoc[Mule Server Notifications]
-  *** xref:notifications-configuration-reference.adoc[Notifications Configuration Reference]
-* xref:using-maven-with-mule.adoc[Maven Support in Mule]
- ** xref:mmp-concept.adoc[Mule Maven Plugin]
- ** xref:package-a-mule-application.adoc[Package a Mule Application]
- ** xref:maven-reference.adoc[Maven Reference]
-* xref:deploying.adoc[Deploy Mule Applications]
- ** xref:deploy-to-cloudhub.adoc[Deploy Applications to CloudHub]
- ** xref:deploy-to-rtf.adoc[Deploy Applications to Runtime Fabric]
- ** xref:deploy-on-premises.adoc[Deploy Applications On-Premises]
-* xref:mule-standalone.adoc[Run Mule Runtime Engine On-Premises]
- ** xref:mule-deployment-model.adoc[On-Premises Deployment Model]
- ** xref:hardware-and-software-requirements.adoc[Hardware and Software Requirements]
- ** xref:runtime-installation-task.adoc[Download and Install Mule]
- ** xref:installing-an-enterprise-license.adoc[Install an Enterprise License]
- ** xref:starting-and-stopping-mule-esb.adoc[Starting and Stopping Mule]
- ** xref:hardening-your-mule-installation.adoc[Hardening Your Mule Installation]
- ** xref:choosing-the-right-clustering-topology.adoc[Cluster Runtime Instances]
-  *** xref:mule-high-availability-ha-clusters.adoc[Mule Runtime HA Cluster Overview]
-  *** xref:creating-and-managing-a-cluster-manually.adoc[Create and Manage a Cluster Manually]
-  *** xref:hadr-guide.adoc[High Availability and Disaster Recovery]
+.xref:index.adoc[Mule Overview]
+* xref:index.adoc[Mule Overview]
+** xref:mule-components.adoc[Mule Components]
+*** xref:about-flows.adoc[Flows and Subflows]
```

Comment:
I'm just wondering if `about-flows.adoc` . needs to be child of `Mule Components` or it could just be at the same level ? Both are part of the Overview and are important pieces.

Reply:
Moved up one level.